webui/Makefile

```diff
@@ -17,15 +17,22 @@ check:
 	$(MAKE) -C elm $@
 	$(MAKE) -C tests $@
 
-get-deps:
-	$(MAKE) -C api-ts-sdk $@
+get-deps: build-api-ts-sdk
```

Comment:
with this change I think we can avoid all the other changes in other make files :thinking:

Reply:
and to the source code

Reply:
I think IF we have this change we'd want to keep the api-ts-dep as part of react's get-deps instead of moving of moving those targets to build and test.